bazel/BUILD

```diff
@@ -308,17 +308,17 @@ selects.config_setting_group(
 )
 
 config_setting(
-    name = "disable_hot_restart",
+    name = "disable_hot_restart_setting",
     values = {"define": "hot_restart=disabled"},
+    visibility = ["//visibility:private"],
```

Comment:
While you are in here can you potentially do this also for the http3 setting to make sure it's always accessed through the aggregate one I added?